fqdn/__init__.py

```diff
@@ -34,10 +34,14 @@ class FQDN:
     def __init__(self, fqdn):
         if not (fqdn and isinstance(fqdn, str)):
             raise ValueError("fqdn must be str")
-        self.fqdn = fqdn
+        self._fqdn = fqdn
```

Comment:
I agree `self.fqdn` is better as `self._fqdn`, and I'll accept that enhancement.